buildSrc/src/main/java/org/elasticsearch/gradle/precommit/ThirdPartyAuditTask.java

```diff
@@ -51,7 +51,7 @@
 public class ThirdPartyAuditTask extends DefaultTask {
 
     private static final Pattern MISSING_CLASS_PATTERN = Pattern.compile(
-        "WARNING: The referenced class '(.*)' cannot be loaded\\. Please fix the classpath!"
+        "WARNING: Class '(.*)' cannot be loaded \\(.*\\)\\. Please fix the classpath!"
```

Comment:
nich catch. Yes this message was changed due to the cleanup of missing class reporting.

Reply:
... a much appreciated change. Thanks for  that !